shapes/profile-shape.ttl

```diff
@@ -0,0 +1,108 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX schema: <http://schema.org/>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX : <#>
+
+:ProfileShape     a sh:NodeShape;
+    dct:title "Profile shape";
+    sh:property 
+            [
+        sh:datatype xsd:string;
+        sh:name "his/hers/theirs...";
+        sh:path solid:preferredRelativePronoun ],
+            [
+        sh:datatype xsd:string;
+        sh:name "him/her/them...";
+        sh:path solid:preferredObjectPronoun ],
+            [
+        sh:datatype xsd:string;
+        sh:name "he/she/they...";
+        sh:path solid:preferredSubjectPronoun ],
```

Comment:
I'd suggest making this all part of a [`sh:group`](https://www.w3.org/TR/shacl/#group) to assist form tools.

My personal opinion is that we should most shapes for client-client standards in Solid should be required to add these [non-validating characteristics](https://www.w3.org/TR/shacl/#nonValidation) so that it is easier for apps to generate nice looking forms for users to supplement and/or fix data that is required for their application from a generic form editor.